### ISSUE TITLE
MSSQL Boot.ps1 stability improvements

### ISSUE THREAD

**Issue by joostmeijles** [sitecoreops/sitecore-images#42]
We improved the stability in the MSSQL `Boot.ps1` script;
- MSSQL server might not be running
- timeouts when attaching databases
- re-attach when db not properly detached

See here for changes: https://github.com/avivasolutionsnl/sitecore-docker/pull/121/files

Do you experience similar issues?

**Comment by pbering** [comment]
It's not something I heard off before but it is still good improvements to have here also. Is it something that you see on every machine?

**Comment by joostmeijles** [comment]
> Is it something that you see on every machine?

Yes, its pretty consistent on various machines. Seems to be related to the performance of a machine; slower machines experience it more often.
